quic-client/src/nonblocking/quic_client.rs

```diff
@@ -229,6 +229,27 @@ pub struct QuicClient {
     stats: Arc<ClientStats>,
 }
 
+const CONNECTION_CLOSE_CODE_APPLICATION_CLOSE: u32 = 0u32;
+const CONNECTION_CLOSE_REASON_APPLICATION_CLOSE: &[u8] = b"dropped";
+
+impl QuicClient {
```

Comment:
Could we implement Drop for QuicClient to make this automatic?

Reply:
No. The await call will be returning LocallyClosed immediately. In any case, I added a unit test test_connection_close. The close is issued after the server is shutdown, I do not see any delay.